scene/resources/packed_scene.cpp

```diff
@@ -2112,6 +2112,24 @@ Ref<SceneState> PackedScene::get_state() const {
 	return state;
 }
 
+bool PackedScene::prefer_compat_format() const {
+	if (state.is_null()) {
+		return true;
+	}
+
+	StringName tile_map_name = "TileMap";
+	StringName tile_map_layer_name = "TileMapLayer";
```

Comment:
```suggestion
	const StringName tile_map_name = "TileMap";
	const StringName tile_map_layer_name = "TileMapLayer";
```
Note that TileMap still saves as PackedIntArray (although it's unintended?), so it's not necessary here as of now.